### DISCUSSION TITLE
Is Redis Cache can connect with TLS ?

### DISCUSSION THREAD

**Discussion by Sarayut-Y** [goravel/goravel#315] [Q&A]
I've research and lookup in  database redis config / goravel-redis 1.0.0, I've never see the option for connect with TLS, It's can be?

<img width="863" alt="Screenshot 2566-11-22 at 12 24 30" src="https://github.com/goravel/goravel/assets/6380878/b6d71e9a-de86-4f1b-bf77-89a4be407a01">

**Comment by hwbrzzl** [comment]
Yes, ot support for now, you can sumit an issue for it.

**Reply by Sarayut-Y** [reply]
Alright :)
